File: 69/69.java
    public static String solution(String line) {
        String[] s = line.split(";");
        String[] s1 = s[0].split(",");
        String[] s2 = s[1].split(",");
        int n = s1.length;
        int m = s2.length;
        int[] a = new int[n];
        int[] b = new int[m];
        for (int i = 0; i < n; i ++) {
            a[i] = Integer.parseInt(s1[i]);
        }
        for (int i = 0; i < m; i ++) {
            b[i] = Integer.parseInt(s2[i]);
        }
        Arrays.sort(a);
        Arrays.sort(b);

        int i = 0, j = 0, cnt = 0;
        while (i < n && j < m) {
            while (j < m && b[j] < a[i]) {
                j ++;
            }
            if (j < m && b[j] >= a[i]) {
                cnt ++;
            }
            i ++;
            j ++;
        }
        if (cnt > n / 2) {
            return "1";
        }
        return "0";
    }

File: 69/69.py
def solution(line):
    a=[int(x) for x in line.split(';')[0].split(',')]
    b=[int(x) for x in line.split(';')[1].split(',')]
    a.sort(reverse=True)
    b.sort(reverse=True)
    l1=len(a)
    l2=len(b)
    j=0
    for i in range(l1):
        if j>=l2 or b[j]<a[i]:
            continue
        j+=1
    if j*2>l1:
        return '1'
    else:
        return '0'

if __name__ == '__main__':
    y = raw_input()
    print(solution(y))

